Added language support to nginx.

m4ktub opened the pull request:
The Nginx configuration is changed to use the accepted languages indicated by the browser and to use a different index html for each language. The first accepted language is used. Rules are also added to allow serving a particular version of the site in a consistent way. For example https://thebitcoincash.fund/en/ will aways serve the English version regardless of the browser.

checksum0 commented:
Before we merge this change, I'd like a language menu to be added to the website. 

Auto-selection from browser is far from good.

m4ktub commented:
This pull request, as is, allows translations to be added and proof-read online at, for example /jp. It has no visible impact on the current site.

So while I agree a language menu, or some sort of language selection, is needed I believe it is needed before the first language is activated (uncommented from the map) and not right now.

Also, I've looked into that for a few minutes and concluded I don't have the design skills to propose a well integrated language menu.

Edit: Removed extra email text. Fixed broken lines.

checksum0 commented:
revert
